contracts/LifCrowdsale.sol

```diff
@@ -109,9 +119,20 @@ contract LifCrowdsale is Ownable, Pausable {
     rate2 = _rate2;
     privatePresaleRate = _privatePresaleRate;
     foundationWallet = _foundationWallet;
-    marketMaker = _marketMaker;
-    minCap = _minCap;
-    maxPresaleWei = _maxPresaleWei;
+  }
+
+  // Set how the rate wei per USD for the public presale, necesary to calculate with more
+  // precision the maxCap on the presale.
+  function setWeiPerUSDinPresale(uint256 _weiPerUSD) onlyOwner {
+    require (block.number < publicPresaleStartBlock && block.number > publicPresaleStartBlock.sub(10));
```

Comment:
So this leaves only the 10 blocks before the presale as the valid time to call this function? I think we should be able to call this function anytime until 10 minutes before the presale start time. The same applies to the next function of course